### ISSUE TITLE
error TS2307: Cannot find module '../../../types/api-output' or its corresponding type declarations.

### ISSUE THREAD

**Issue by HarryLu33** [TEAMMATES/teammates#12602]
error TS2307: Cannot find module '../../../types/api-output' or its corresponding type declarations.
- **Environment**: 
master branch at commit 62139bc8245b6fe00012a912e653a6b183391b00
**Steps to reproduce**
Run src\web\app\components\comment-box\comment-edit-form\comment-edit-form.component.spec.ts file. 

**Expected behaviour**
Compile successfully

**Actual behaviour**
`Test suite failed to run

    src/web/app/components/teammates-common/feedback-path.pipe.ts:2:41 - error TS2307: Cannot find module '../../../types/api-output' or its corresponding type declarations.

    2 import { FeedbackParticipantType } from '../../../types/api-output';
                                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~`


**Additional info**
<!-- You can enter any additional relevant info here. -->
<!-- If you know what caused the bug and/or how to resolve the bug, you can include them in this section. -->
I am currently working on issue #12588 to Improve the test code coverage of CommentEditFormComponent. When I tried to run the test, the code could not compile. It shows cannot find module '../../../types/api-output'. I am new to this project, is there anything I missed to get this error?

**Comment by jasonqiu212** [comment]
Hi @HarryLu33, Have you tried running the `./gradlew createConfigs` command to generate the config files? For more information, please refer to our [Setting Up](https://teammates.github.io/teammates/setting-up.html) documentation page :)

**Comment by HarryLu33** [comment]
yes, I have gone through all the setup steps. However the types/api-output directory is still missing

**Comment by cedricongjh** [comment]
hi @HarryLu33 I think what you should be missing is gradLew `generateTypes`, do try that out and let us know if it works!
